content/en/examples/application/basic-daemonset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: example-daemonset
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: example
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: example
+    spec:
+      containers:
+      - name: pause
+        image: registry.k8s.io/pause
+      initContainers:
+      - name: log-machine-id
+        image: busybox
```

Comment:
nit: consider pinning to a specific version of this container. It's moot because `cat` and `sh` are defined in POSIX.